package.json

```diff
@@ -53,6 +53,8 @@
     "babel-loader": "^8.0.5",
     "clean-webpack-plugin": "^2.0.1",
     "css-loader": "^2.1.1",
+    "enzyme": "^3.9.0",
```

Comment:
Do we really need enzyme? Was there a particular issue that couldn't be resolved with react-testing-library alone?